ghost.js

```diff
@@ -0,0 +1,7 @@
+var Ghost = function() {};
+
+Ghost.prototype.coloring = function() {
+  var colors = ["red", "blue", "yellow"];
```

Comment:
other than implementing three colors (one of which isn't in the specification), the code looks good.